content/client-server-api/modules/event_replacements.md

```diff
@@ -211,16 +205,35 @@ event that is the target of an `m.replace` relationship. For example:
 ```json
 {
   "event_id": "$original_event_id",
-  // irrelevant fields not shown
+  "type": "m.room.message",
```

Comment:
fwiw, I'm still very sad that https://github.com/matrix-org/matrix-spec/issues/688 isn't a thing, which would ideally make this sort of example easy to update in the future.

Non-blocking for this review.